src/grow_worker/worker.py

```diff
@@ -40,30 +53,65 @@ def run_mesido(input_esdl: str) -> str:
     influxdb_host = os.environ.get("INFLUXDB_HOSTNAME", "localhost")
     influxdb_port = int(os.environ.get("INFLUXDB_PORT", "8086"))
 
-    logger.info("Will write result profiles to influx: {}. At {}:{}",
-                write_result_db_profiles,
-                influxdb_host,
-                influxdb_port)
-
-    solution: GROWProblem = mesido_func(
-        mesido_workflow,
-        base_folder=base_folder,
-        esdl_string=base64.encodebytes(input_esdl.encode("utf-8")),
-        esdl_parser=ESDLStringParser,
-        write_result_db_profiles=write_result_db_profiles,
-        influxdb_host=influxdb_host,
-        influxdb_port=influxdb_port,
-        influxdb_username=os.environ.get("INFLUXDB_USERNAME"),
-        influxdb_password=os.environ.get("INFLUXDB_PASSWORD"),
-        influxdb_ssl=False,
-        influxdb_verify_ssl=False,
-        update_progress_function=None,
-        profile_reader=InfluxDBProfileReader,
+    logger.info(
+        "Will write result profiles to influx: {}. At {}:{}",
+        write_result_db_profiles,
+        influxdb_host,
+        influxdb_port,
     )
+    try:
+        solution: GROWProblem = mesido_func(
+            mesido_workflow,
+            base_folder=base_folder,
+            esdl_string=base64.encodebytes(input_esdl.encode("utf-8")),
+            esdl_parser=ESDLStringParser,
+            write_result_db_profiles=write_result_db_profiles,
+            influxdb_host=influxdb_host,
+            influxdb_port=influxdb_port,
+            influxdb_username=os.environ.get("INFLUXDB_USERNAME"),
+            influxdb_password=os.environ.get("INFLUXDB_PASSWORD"),
+            influxdb_ssl=False,
+            influxdb_verify_ssl=False,
+            update_progress_function=None,
+            profile_reader=InfluxDBProfileReader,
+        )
+    except SystemExit as e:
```

Comment:
This is only triggered when mesido throws the SystemExit as it runs in a subprocess